### PR TITLE
Allow upgrading to specific version using `create-liveblocks-app --upgrade`

### PR DESCRIPTION
**Pull request by nvie**
To upgrade all Liveblocks packages to the latest version:

    $ npx create-liveblocks-app@latest --upgrade

To upgrade all Liveblocks packages to the `beta` version:

    $ npx create-liveblocks-app@latest --upgrade beta

To upgrade all Liveblocks packages to a specific version:

    $ npx create-liveblocks-app@latest --upgrade 1.12.0

### CODE REVIEW

**Review by marcbouchenoire — COMMENTED** [review]
I think stylistically I would prefer `--upgrade` (latest) and `--upgrade=1.12.0`/`--upgrade="beta"` given that it's already a flag and not a command, would it be weird?

**Comment by nvie** [comment]
> I think stylistically I would prefer `--upgrade` (latest) and `--upgrade=1.12.0`/`--upgrade="beta"` given that it's already a flag and not a command, would it be weird?

@marcbouchenoire I tried, but I could not get it to work. A flag is either a boolean, or it's a string, in which case it takes an argument, but I couldn't get it to work with the cmdline arg parser to do this hybrid form, so I settled on this optional `tag` flag, which I didn't mind too much. If you have an idea of how to implement the hybrid flag, I'm still open to it.

File: packages/create-liveblocks-app/scripts/demo.js [hunk 1]
```diff
@@ -45,7 +43,7 @@ fs.rmSync(testDir, { recursive: true, force: true });
 fs.mkdirSync(testDir);
 fs.writeFileSync(path.join(testDir, "package.json"), pkgJson);
 
-execSync(`node ${esbuildOptions.outfile} ${flags}`, {
+execSync(`node ${esbuildOptions.outfile} ${process.argv.slice(2)}`, {
```
Comment (nvie) [inline packages/create-liveblocks-app/scripts/demo.js:46]: What's happening here? This feels a bit brittle.

Comment (marcbouchenoire) [reply packages/create-liveblocks-app/scripts/demo.js:46]: This is just a demo script to test the built CLI locally, `npm run demo` etc. This change forwards all flags (the two first args are `node` and the file path) so we can do `npm run demo --upgrade` with it.

**Review by CTNicholas — APPROVED** [review]
Hey guys, this had a few bugs, so I've just fixed them. You can use these commands:

```bash
# latest
--upgrade

## 2.0.0
--upgrade 2.0.0

# Also works
--upgrade @2.0.0
```

If you use the UI it will always upgrade to the latest version.

**Pull request merged**